src/pages/App.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react';
+import logo from './logo.svg';
+import '../styles/App.scss';
+import { FrontEndMentorHeader } from '../components/FrontEndMentor';
+import { Filter } from '../components/Filter';
+import { Roadmap } from '../components/Roadmap';
+import { ToolBar } from '../components/ToolBar';
+import { RequestCards } from '../components/RequestCards';
+import { data } from 'data';
+import { ProductRequest } from 'types';
+
+const App = () => {
+  const [allData, setAllData] = useState(data);
+  const [showData, setShowData] = useState<ProductRequest[]>(
+    allData.productRequests,
+  );
+
+  const { productRequests } = allData;
+
+  return (
+    <div className="App">
+      <div className="App-col one">
+        <FrontEndMentorHeader />
+        <Filter
+          productRequests={productRequests}
+          showData={showData}
+          setShowData={setShowData}
+        />
+        <Roadmap />
+      </div>
+      <div className="App-col">
```

Comment:
How about this tag. We can get more semantic here instead of a `div`.

Reply:
Also changed to section.

Reply:
Perhaps another one? https://www.w3schools.com/html/html5_semantic_elements.asp

Reply:
ah, should it be article since it contains comments?

Reply:
oooh that does make sense. changed it